### PR TITLE
Add integration tests for TransXChange parsing

### PR DESCRIPTION
**Pull request by woshilapin**

### CODE REVIEW

File: fixtures/transxchange2ntfs/input/config.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+	"contributor": {
+		"contributor_id": "UK",
```
Comment (datanel) [inline fixtures/transxchange2ntfs/input/config.json]: is it a tab or 8 spaces?

Comment (woshilapin) [reply fixtures/transxchange2ntfs/input/config.json]: It's tabulations, why?

Comment (datanel) [reply fixtures/transxchange2ntfs/input/config.json]: we use 4 spaces for all

Comment (woshilapin) [reply fixtures/transxchange2ntfs/input/config.json]: This is a copy from https://github.com/CanalTP/transit_model/blob/master/fixtures/kv12ntfs/config.json which also contains tabulations. Should I change it everywhere?

Comment (datanel) [reply fixtures/transxchange2ntfs/input/config.json]: yes please.

Comment (woshilapin) [reply fixtures/transxchange2ntfs/input/config.json]: Done in f0214eb25ee98a81b04edcc11762b464668340cc.

**Pull request merged**